### PR TITLE
Feature/stackn 181

### PR DESCRIPTION
**Pull request by stefanhellander**
## Status

- [X] Ready
- [ ] Draft
- [ ] Hold

## Description

Adds option to create project on a specific cluster. Notebooks will be created on the same cluster.

## Types of changes

What types of changes does your code introduce to STACKn?

- [ ] Hotfix (fixing a critical bug in production)
- [ ] Bugfix
- [X] New feature
- [ ] Documentation update

## Checklist

_If you're unsure about any of the items below, don't hesitate to ask. We're here to help! 
This is simply a reminder of what we are going to look for before merging your code._

- [ ] This pull request is against **develop** branch (not applicable for hotfixes)
- [ ] I have included a link to the issue on GitHub or JIRA (if any)
- [ ] I have included migration files (if there are changes to the model classes)
- [ ] I have read the [CONTRIBUTING](https://github.com/scaleoutsystems/stackn/blob/master/CONTRIBUTING.md) doc
- [ ] I have included tests to complement my changes
- [ ] I have updated the related documentation (if necessary) 
- [ ] I have updated the release notes (docs/releasenotes.md)
- [ ] I have added a reviewer for this pull request
- [ ] I have added myself as an author for this pull request

## Further comments

Anything else you think we should know before merging your code!

### CODE REVIEW

**Review by dstoyanova — APPROVED** [review]

**Comment by dstoyanova** [comment]
@stefanhellander Add info to the release notes `docs/releasenotes.md`

File: components/studio/labs/models.py [hunk 1]
```diff
@@ -127,9 +139,11 @@ def pre_save_labs(sender, instance, using, **kwargs):
         extraVolumes = ""
         extraVolumeMounts = ""
         i = 1
+        print("VOLS")
+        print(vols)
```
Comment (morganekmefjord) [inline components/studio/labs/models.py:143]: Remove these I guess?

**Review by morganekmefjord — APPROVED** [review]
Great implementation.

**Comment by stefanhellander** [comment]
This branch has been selectively merged into develop, and some parts of this should probably be integrated in a different way now (multi-cluster setup). I will close this PR without merging.

**Pull request closed without merging**